packages/tables/src/Table/Concerns/CanSearchRecords.php

```diff
@@ -77,4 +79,16 @@ public function getColumnSearchIndicators(): array
     {
         return $this->getLivewire()->getTableColumnSearchIndicators();
     }
+
+    public function searchPlaceholder(string | Closure $searchPlaceholder): static
+    {
+        $this->searchPlaceholder = $searchPlaceholder;
+
+        return $this;
+    }
+
+    public function getSearchPlaceholder(): ?string
+    {
+        return $this->evaluate($this->searchPlaceholder);
```

Comment:
We might want to move the default translation in here?

Reply:
Not sure. There are quite a lot of translations inside the views. There is even another translation in the same view file, but none in the `Concerns/CanSearchRecords.php` class. And you when moving it from the view, there is no default value for the view prop `placeholder` anymore. Not sure if this is better. But if you want, I can change it?